ppc64le: osbuild-tests failure

From RHEL 8 on ppc64le:

```
TASK [Run osbuild-tests test] **************************************************
Friday 29 May 2020  12:55:02 -0400 (0:00:00.670)       0:10:51.753 ************ 
fatal: [localhost]: FAILED! => changed=true 
  cmd:
  - /usr/libexec/tests/osbuild-composer/osbuild-tests
  - -test.v
  delta: '0:00:22.795913'
  end: '2020-05-29 12:55:25.418691'
  msg: non-zero return code
  rc: 1
  start: '2020-05-29 12:55:02.622778'
  stderr: ''
  stderr_lines: <omitted>
  stdout: |-
    === RUN   TestComposeCommands
    --- FAIL: TestComposeCommands (0.71s)
        main_test.go:257:
                    Error Trace:    main_test.go:257
                                                            main_test.go:40
                    Error:          Received unexpected error:
                                    exit status 1
                    Test:           TestComposeCommands
                    Messages:       Command failed: exit status 1
    === RUN   TestEverything
    --- PASS: TestEverything (21.18s)
    === RUN   TestSourcesCommands
    --- PASS: TestSourcesCommands (0.90s)
    FAIL
  stdout_lines: <omitted>
```

--- a/cmd/osbuild-tests/main_test.go
+++ b/cmd/osbuild-tests/main_test.go
@@ -46,7 +46,7 @@ func TestComposeCommands(t *testing.T) {
 	runComposer(t, "compose", "list", "failed")
 
 	// Full integration tests
-	uuid := buildCompose(t, "empty", "ami")
+	uuid := buildCompose(t, "empty", "qcow2")
 	defer deleteCompose(t, uuid)
 
 	runComposer(t, "compose", "info", uuid.String())
@@ -73,9 +73,13 @@ func TestComposeCommands(t *testing.T) {
 	defer os.Remove(uuid.String() + "-logs.tar")
 
 	runComposer(t, "compose", "image", uuid.String())
-	_, err = os.Stat(uuid.String() + "-image.vhdx")
-	require.NoError(t, err, "'%s-image.vhdx' not found", uuid.String())
-	defer os.Remove(uuid.String() + "-image.vhdx")
+	_, err = os.Stat(uuid.String() + "-disk.qcow2")
+	require.NoError(t, err, "'%s-disk.qcow2' not found", uuid.String())
+	defer os.Remove(uuid.String() + "-disk.qcow2")
+
+	// Check that reusing the cache works ok
+	uuid = buildCompose(t, "empty", "qcow2")
+	defer deleteCompose(t, uuid)
 
 	// https://github.com/osbuild/osbuild-composer/pull/180
 	// uuid = startCompose(t, "empty", "tar")

